namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/domain-events-queue-bvls.tf

```diff
@@ -113,7 +113,8 @@ resource "aws_sns_topic_subscription" "hmpps_book_a_video_link_domain_subscripti
       "book-a-video-link.video-booking.cancelled",
       "book-a-video-link.appointment.created",
       "prisoner-offender-search.prisoner.released",
-      "prison-offender-events.prisoner.merged"
+      "prison-offender-events.prisoner.merged",
+      "whereabouts-api.videolink.migrate"
     ]
```

Comment:
Whereabouts produces this event - it doesn't need to subscribe to listen to it??